### PR TITLE
Sync v2-7-stable with v2-7-test to release 2.7.2

### PR DESCRIPTION
**Pull request by ephraimbuddy**
Time for `2.7.2rc1`!

### CODE REVIEW

File: RELEASE_NOTES.rst [hunk 1]
```diff
@@ -21,6 +21,117 @@
...
+- Fix FAB-related logging format interpolation (#34139)
+- Fix query bug in ``next_run_datasets_summary`` endpoint (#34143)
+- Fix for TaskGroup toggles for duplicated labels (#34072)
+- Fix the required permissions to clear a TI from the UI (#34123)
+- Reuse ``_run_task_session`` in mapped ``render_template_fields`` (#33309)
+- Fix scheduler logic to plan new dag runs by ignoring manual runs (#34027)
+- Add missing audit logs for Flask actions add, edit and delete (#34090)
+- Hide Irrelevant Dag Processor from Cluster Activity Page (#33611)
+- Remove infinite animation for pinwheel, spin for 1.5s (#34020)
+- Restore rendering of provider configuration with ``version_added`` (#34011)
```
Comment (eladkal) [inline RELEASE_NOTES.rst:62]: this is internal change is it not?

File: RELEASE_NOTES.rst [hunk 2]
```diff
@@ -21,6 +21,117 @@
...
+
+Significant Changes
+^^^^^^^^^^^^^^^^^^^
+
+No significant changes
+
+
+Bug Fixes
+"""""""""
+- Import ``AUTH_REMOTE_USER`` from FAB in WSGI middleware example (#34721)
```
Comment (eladkal) [inline RELEASE_NOTES.rst]: this is doc only change

File: RELEASE_NOTES.rst [hunk 3]
```diff
@@ -21,6 +21,117 @@
...
+- Fixed rows count in the migration script (#34348)
```
Comment (eladkal) [inline RELEASE_NOTES.rst]: We can replace this with a more informative note user added in the newsfragment

`Fixed AttributeError: 'Select' object has no attribute 'count' during the airflow db migrate command`

File: RELEASE_NOTES.rst (hunk 1 above)
Comment (ephraimbuddy) [reply RELEASE_NOTES.rst:62]: It's a bug: https://github.com/apache/airflow/issues/34005

**Review by eladkal — APPROVED** [review]

**Review by hussein-awala — APPROVED** [review]

**Review by pierrejeambrun — APPROVED** [review]

**Comment by potiuk** [comment]
The failure in "pull request target" can be ignored. All the remaining steps in "pull request target" workflow are generally ignored when PR is run from "apache" repository (but in this case the pipx/packaging issue causes even the "build info" job to fail there - normally the output of that build info job is used to skipp all the other jobs in "pull request target" workflow, so you can merge it @ephraimbuddy - ignoring the "Pull request target" "build info" workflow (cc: @hussein-awala ).

**Review by potiuk — APPROVED** [review]

**Comment by ephraimbuddy** [comment]
> The failure in "pull request target" can be ignored. All the remaining steps in "pull request target" workflow are generally ignored when PR is run from "apache" repository (but in this case the pipx/packaging issue causes even the "build info" job to fail there - normally the output of that build info job is used to skipp all the other jobs in "pull request target" workflow, so you can merge it @ephraimbuddy - ignoring the "Pull request target" "build info" workflow (cc: @hussein-awala ).

Ok. Thanks for info

**Pull request merged**